packages/suite/src/views/wallet/coinmarket/p2p/offers/Avatar.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { SuiteThemeColors } from '@trezor/components';
+import { resolveStaticPath } from '@trezor/utils';
+import { OnlineStatus } from 'invity-api';
+
+const Wrapper = styled.div`
+    width: 20px;
+    height: 20px;
+    position: relative;
+    margin-right: 4px;
+`;
+
+const Image = styled.img`
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+`;
+
+interface StatusIndicatorProps {
+    onlineStatus: OnlineStatus;
+}
+
+const getStatusColor = (onlineStatus: OnlineStatus, theme: SuiteThemeColors) => {
+    const statusColors = {
+        ONLINE: theme.TYPE_GREEN,
+        RECENTLY_ONLINE: theme.TYPE_ORANGE,
+        OFFLINE: theme.TYPE_LIGHT_GREY,
+    };
+
+    return statusColors[onlineStatus];
+};
+
+const StatusIndicator = styled.div<StatusIndicatorProps>`
+    position: absolute;
+    top: 0;
+    left: 14px;
+    width: 5px;
+    height: 5px;
+    border: 3px solid ${({ onlineStatus, theme }) => getStatusColor(onlineStatus, theme)};
+    border-radius: 50%;
+`;
+
+interface AvatarProps {
+    onlineStatus: OnlineStatus;
+}
+
+export const Avatar = ({ onlineStatus }: AvatarProps) => (
+    <Wrapper>
+        <Image src={resolveStaticPath('/images/svg/coinmarket-avatar.svg')} />
```

Comment:
Please use the Image component from `@trezor/components` to keep it consistent with all other images...

Reply:
Done.

Reply:
to make it really consistent it means to define `COINMARKET_AVATAR` in `SVG_IMAGES` and use it as `<Image image="COINMARKET_AVATAR" />`... see for example usage of `COINMARKET_SUCCESS` image...

Reply:
I see... Working on it!